### PR TITLE
[Estuary][PVR] PVR Info dialog: Add 'Add timer' button.

### PR DESCRIPTION
**Pull request by ksooo**
According to several forum posts it seems that we have many users unable to find Kodi's series recordings functions, because they do not find the respective context menu, for example in the pvr guide window. :-/ All they are able to do is to hit enter/ok/select after selecting an epg event, getting then to the pvr info dialog currently containing only "record", but no "add timer" button. ;-)

This PR adds the so often missed "Add timer" button to the pvr info dialog (core & estuary).
screenshot002

@Jalle19 for the code review?
@bignoid for the skin changes?
@MartijnKaijser would like to have this in beta 3, but as it is not literally a bug fix, it's up to you to decide

### CODE REVIEW

File: xbmc/pvr/dialogs/GUIDialogPVRGuideInfo.cpp [hunk 1]
```diff
@@ -123,6 +137,21 @@ bool CGUIDialogPVRGuideInfo::OnClickButtonRecord(CGUIMessage &message)
   return bReturn;
 }
 
+bool CGUIDialogPVRGuideInfo::OnClickButtonAddTimer(CGUIMessage &message)
+{
+  bool bReturn = false;
+
+  if (message.GetSenderId() == CONTROL_BTN_ADD_TIMER)
+  {
+    if (!m_progItem->Timer())
```
Comment (Jalle19) [inline xbmc/pvr/dialogs/GUIDialogPVRGuideInfo.cpp]: Is `m_progItem` guaranteed to not be NULL?

**Comment by Jalle19** [comment]
+1 for having this in Krypton, it's a pretty isolated change. Estuary is the new default skin so introducing it with bad discoverability could be considered a bug.

File: xbmc/pvr/dialogs/GUIDialogPVRGuideInfo.cpp (hunk 1 above)
Comment (ksooo) [inline xbmc/pvr/dialogs/GUIDialogPVRGuideInfo.cpp]: not 100% sure; so lets put in a nullptr check here, like done in other places in this file.

Comment (Jalle19) [inline xbmc/pvr/dialogs/GUIDialogPVRGuideInfo.cpp]: Good idea

Comment (ksooo) [inline xbmc/pvr/dialogs/GUIDialogPVRGuideInfo.cpp]: done

**Comment by MartijnKaijser** [comment]
Go for it

**Comment by ronie** [comment]
could you cherry-pick https://github.com/ronie/xbmc/commit/db22664f64afcb2923cb4e84af3b4216e58b2798

**Comment by BigNoid** [comment]
Looks good to me with the timer icon from @ronie

**Comment by ksooo** [comment]
@ronie thanks a lot for the icon.

**Comment by ksooo** [comment]
jenkins build this please

**Comment by ksooo** [comment]
jenkins build ths please

**Comment by MartijnKaijser** [comment]
Jenkins build this please

**Comment by ksooo** [comment]
OSX64 jenkins error is unrelated. 
<pre>
/Users/Shared/jenkins/workspace/OSX-64/tools/buildsteps/osx64/run-tests: line 11: 16044 Segmentation fault: 11  build/kodi-test --gtest_output=xml:gtestresults.xml
</pre>

**Pull request merged**